recipes/SimpleElastix/bld.bat

```diff
@@ -0,0 +1,4 @@
+
+
+
+if errorlevel 1 exit 1
```

Comment:
So, in the handful of cases where I have tried Windows, I have gone for the `nmake` strategy. That's what I tried to do for `protobuf`. Here is the [`bld.bat`]( https://github.com/conda-forge/protobuf-feedstock/blob/master/recipe/bld.bat ). Though there has been some [Windows black magic]( https://github.com/conda-forge/boost-feedstock/blob/master/recipe/bld.bat#L1-L15 ) that I have seen in a few cases, which is a mess, but maybe you could give it a try or someone could help you with it. Also, this [wiki]( https://github.com/conda/conda/wiki/VC-features ) is pretty handy.